Clean up and Update val-2b (#202)
idaholab/TMAP8: doc/content/verification_and_validation/val-2b.md

## Test Description

This validation problem is taken from [!cite](macaulay1991deuterium) and is part of the validation suite of TMAP4 and TMAP7 [!citep](longhurst1992verification,ambrosek2008verification). We here reproduce, with some updates, the case modeled in [!cite](longhurst1992verification,ambrosek2008verification).

R.G. Macaulay-Newcombe et al. conducted thermal absorption and desorption experiments, as well as implantation experiments, on wafers of polished beryllium.
Of the several data sets presented, the one modeled here is run 2a1 represented in Figure 2(a) in their publication. The beryllium sample was 0.4 mm thick and had an area of 104 mm$^2$, as illustrated in [val-2b_schematic].

> It was polished to a mirror finish and then exposed to 13.3 kPa of deuterium at 773 K for 50 hours. It was quickly cooled under a vacuum of about 1 $\mu$Pa. The cooling time constant for the apparatus is taken as 45 minutes. After removing the sample from the charging furnace, it was transferred in the air to a thermal desorption furnace where the temperature was increased from ambient (300 K) to 1073 K at the rate of 3 K/min. This was done under vacuum, and the pressure of the chamber was monitored by the residual gas analysis and calibrated against standard leaks. In that way, the emission rate from the sample could be measured as a function of temperature. The sample pressure and temperature histories are shown in [val-2b_temperature_pressure_history].

Author: > Where do you see the cooling time constant in the Macaulay-Newcombe paper? The discussion of the parameters for this sample in their paper (page 2, paragraphs 3 and 4) doesn't mention it, and neither does the discussion in Section 3.2.

It is not specified. This is an assumption first made in the TMAP4 case. I'll specify this here. 

> The cooling period is also just short enough to get to ambient temperature, but is much shorter than the timeframe considered in the paper (>24h between removal from the charging furnace to the ramp up in the desorption furnace). How do you square this?

Alright, great question - I read the original paper 50 times and tried to make sense of what was described, but I'm still quite unsure about the experimental procedure. I couldn't figure out if the baking, cooldowns, and rest procedures were applied to the sample itself or the "empty" chamber to clean up any impurities and prepare it for the experiment. In parallel, TMAP4/TMAP7 used 40 minutes, which was not enough time to decrease the temperature to the initial temperature of the desorption phase (300 K). To me, it wouldn't make sense to start the head up at 300 K if the sample itself was not yet at 300 K, so I picked a time that was long enough to bring the temperature of the sample to around 300K, but did not increase the length of the history unnecessarily (nothing significant happens at 300 K because the kinetics are very slow). 

I can be more transparent about the uncertainty in the history in the documentation.

> It was polished to a mirror finish and then exposed to 13.3 kPa of deuterium at 773 K for 50 hours. It was quickly cooled under a vacuum of about 1 $\mu$Pa. The cooling time constant for the apparatus is taken as 45 minutes. After removing the sample from the charging furnace, it was transferred in the air to a thermal desorption furnace where the temperature was increased from ambient (300 K) to 1073 K at the rate of 3 K/min. This was done under vacuum, and the pressure of the chamber was monitored by the residual gas analysis and calibrated against standard leaks. In that way, the emission rate from the sample could be measured as a function of temperature. The sample pressure and temperature histories are shown in [val-2b_temperature_pressure_history].

Author: Done

Experimental data from that measurement, given in Figure 2 (a) in [!cite](macaulay1991deuterium) are reproduced in [val-2b_comparison].

!media figures/val-2b_schematic.jpg
    style=width:75%;margin-bottom:2%;margin-left:auto;margin-right:auto
    id=val-2b_schematic
    caption=Schematic of the experiment setup (figure elements are not to scale).

!media comparison_val-2b.py
    image_name=val-2b_temperature_pressure_history.png
    style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
    id=val-2b_temperature_pressure_history
    caption=Pressure and temperature histories.

From Rutherford backscattering measurements made on the samples before charging with deuterium, they deduced that the thickness of the oxide film was 18 nm. This is typical for polished beryllium. The metal is so reactive in air that the film forms almost immediately after any surface oxide removal. On the other hand, it is relatively stable and would only grow slightly when exposed to air between charging and thermal desorption.

> This experiment is modeled using a two-segment model in TMAP8 with the segments linked. The first segment is the BeO film which is modeling using 18 elements each 1 nm in length. The second segment is the beryllium with reflective boundary condition at the mid-plane. The beryllium segment is modeled using 40 elements each 50 $\mu$m thick. The solubility of deuterium in beryllium used was that given by [!cite](wilson1990beryllium) based on the work done by W. A. Swansiger also of Sandia National Laboratory.

Reviewer: ```suggestion
This experiment is modeled using a two-segment model in TMAP8 with the segments linked. The first segment is the BeO film which is modeled using 18 elements, each 1 nm in length. The second segment is the beryllium with reflective boundary condition at the mid-plane. The beryllium segment is modeled using 40 elements, each 50 $\mu$m thick. The solubility of deuterium in beryllium used was that given by [!cite](wilson1990beryllium) at Sandia National Laboratory (SNL) based on the work done by W. A. Swansiger, also of SNL.
```

The diffusivity of deuterium in beryllium was measured by [!cite](abramov1990deuterium). They made measurements on high-grade (99$\%$ pure) and extra-grade (99.8$\%$ pure). The values used here are those for high-grade beryllium, consistent with Dr. Macaulay-Newcombe's measurements of the purity of his samples.

> Deuterium transport properties of the BeO are more challenging. First, it is not clear in which state the deuterium exists in the BeO. However, it has been observed [!cite](longhurst1990tritium) that an activation energy of -78 kJ/mol (exothermic reaction) is evident for tritium coming out of neutron irradiated beryllium in work done by D. L. Baldwin of Battelle Pacific Northwest Laboratory. The same value of energy has appeared in other results (can be inferred from Dr. Swansiger's work cited by [!cite](wilson1990beryllium) and by [!cite](causey1990tritium), among others), so one may be justified in using it. Concerning the solubility, measurements reported by [!cite](macaulay1992thermal) and in follow-up conversations indicate about 200 appm of D in BeO after exposure to 13.3 kPa fo D$_2$ at 773 K. That suggests a coefficient of only 1.88 x 10$^{18}$ d/m$^3$Pa$^{1/2}$. Since much of the deuterium in the oxide layer will get out during the cool-down process (and because it gives a good fit) the solubility coefficient is taken to be 5 $\times$ 10$^{20}$ d/m$^3$Pa$^{1/2}$.

Reviewer: ```suggestion
Deuterium transport properties of the BeO are more challenging. First, it is not clear in which state the deuterium exists in the BeO. However, it has been observed [!cite](longhurst1990tritium) that an activation energy of -78 kJ/mol (exothermic reaction) is evident for tritium coming out of neutron-irradiated beryllium in work done by D. L. Baldwin of Pacific Northwest Laboratory. The same value of energy has appeared in other results (can be inferred from Dr. Swansiger's work cited by [!cite](wilson1990beryllium) and by [!cite](causey1990tritium), among others), so one may be justified in using it. Concerning the solubility, measurements reported by [!cite](macaulay1992thermal) and in follow-up conversations indicate about 200 appm of D in BeO after exposure to 13.3 kPa of D$_2$ at 773 K. That suggests a coefficient of only 1.88 x 10$^{18}$ d/m$^3$Pa$^{1/2}$. Since much of the deuterium in the oxide layer will get out during the cool-down process (and because it gives a good fit), the solubility coefficient is taken to be 5 $\times$ 10$^{20}$ d/m$^3$Pa$^{1/2}$.
```

"Batelle Pacific Northwest Laboratory" is a tautology (then-PNL has only ever been managed by Batelle, who named it PNL).

> Deuterium diffusion measurements in BeO were made by [!cite](fowler1977tritium). They found a wide range of results for diffusivity in BeO, depending on the physical form of the material, having measured it for single-crystal, sintered, and powdered BeO. The model in [!citep](longhurst1992verification,ambrosek2008verification) uses one expression for the charging phase and another for the thermal desorption phase, believing that the surface film changed somewhat during the transfer between the two furnaces. For the charging phase diffusivity, the model uses 20 times that for the sintered BeO. Thermal expansion mismatches tend to open up cracks and channels in the oxide layer, so this seems a reasonable value. The same activation energy of 48.5 kJ/mol, is retained, however. For the thermal desorption phase, the diffusivity prefactor of the sintered material (7x10$^{-5}$ m$^2$/sec) and an activation energy of 223.7 kJ/mol (53.45 kcal/mol) are used. These values give good results and lie well within the scatter of Fowler's data. Exposure of the sample to air after heating should have made the oxide more like a single crystal by healing the cracks that may have developed. Diffusivities and solubilities used in the simulation are listed in [val-2b_parameters].

Reviewer: ```suggestion
Deuterium diffusion measurements in BeO were made by [!cite](fowler1977tritium). They found a wide range of results for diffusivity in BeO depending on the physical form of the material, having measured it for single-crystal, sintered, and powdered BeO. The model in [!citep](longhurst1992verification,ambrosek2008verification) uses one expression for the charging phase and another for the thermal desorption phase, believing that the surface film changed somewhat during the transfer between the two furnaces. For the charging phase diffusivity, the model uses 20 times that for the sintered BeO. Thermal expansion mismatches tend to open up cracks and channels in the oxide layer, so this seems a reasonable value. The same activation energy of 48.5 kJ/mol, is retained, however. For the thermal desorption phase, the diffusivity prefactor of the sintered material (7x10$^{-5}$ m$^2$/sec) and an activation energy of 223.7 kJ/mol (53.45 kcal/mol) are used. These values give good results and lie well within the scatter of Fowler's data. Exposure of the sample to air after heating should have made the oxide more like a single crystal by healing the cracks that may have developed. Diffusivities and solubilities used in the simulation are listed in [val-2b_parameters].
```

The experimental data used in this case comes directly from Figure (2) in [!cite](macaulay1991deuterium). This is in contrast with the data used in [!cite](longhurst1992verification,ambrosek2008verification), which, although the scale of the data is very similar, differs slightly. Note also that the units in Figure (2) from [!cite](macaulay1991deuterium) should be atoms/mm$^2$/s $\times 10^{10}$ instead of atoms/mm$^2$ $\times 10^{10}$, which is corrected in [val-2b_comparison].

To verify that the solubility ratio at the interface between the beryllium and its oxide is appropriately modeled, [val-2b_ratio] compares the known solubility ratio with the calculated deuterium concentration ratio at the interface, and they match, as expected.

!media comparison_val-2b.py
       image_name=val-2b_ratio.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=val-2b_ratio
       caption=Solubility and concentration ratio at the Be-BeO interface. The match shows that the solubility difference is properly enforced in TMAP8.

## Input files

!style halign=left
The input file for this case can be found at [/val-2b.i].

> To limit the computational costs of the test cases (test case are expected to run in under 2 seconds), the tests in [/val-2b/tests] run a version of the files with a shorter history, looser tolerance, and larger time step. More information about the changes can be found in the test specification file for this case, namely [/ver-1d/tests].

Reviewer: ```suggestion
To achieve short regression tests (under 2 seconds walltime), the tests in [/val-2b/tests] run a version of the files with a shorter history, looser tolerance, and larger time step. More information about the changes can be found in the test specification file for this case, namely [/ver-1d/tests].
```
